doc/en/parametrize.rst

```diff
@@ -81,17 +81,23 @@ them in turn:
 
 .. note::
 
-    pytest by default escapes any non-ascii characters used in unicode strings
-    for the parametrization because it has several downsides.
-    If however you would like to use unicode strings in parametrization and see them in the terminal as is (non-escaped), use this option in your ``pytest.ini``:
+    1. Parameter values are passed as-is to tests (no copy whatsoever), so if you mutate them,
+       the mutations will be reflected in subsequent test case calls.
```

Comment:
Another suggestion upon reading again, to clarify a bit:

```suggestion
    1. Parameter values are passed as-is to tests (no copy whatsoever), so if for example you use a list or a dict as a paramater value and mutate it,
       the mutations will be reflected in subsequent test case calls.
```

Reply:
Much better,thanks!

Reply:
Just committed the example as a separate paragraph.